haiku-games/lightsoff/lightsoff-1.1.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A puzzle game based on the original Lights Out!"
+DESCRIPTION="LightsOff is a rendition of the addictive puzzle game 'Lights \
+Out!' which was developed by Tiger Electronics for a handheld game console \
+in the 1990s.
+
+On a square grid some tiles are lit, some aren't. The objective is to turn \
+all the lights off by clicking the fewest number of tiles. What makes it \
+challenging: clicking a tile also toggles the tiles to left/right and \
+above/below it."
+APP="LightsOff"
+instDir=$appsDir/$APP
+HOMEPAGE="https://github.com/HaikuArchives/$APP"
+COPYRIGHT="2005 DarkWyrm
+	2018 Owen Pan"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="868f3d84562d51350338a708db7466d1ac40e883ccae237b3a383a411e8365e6"
+SOURCE_DIR="$APP-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	lightsoff = $portVersion
+	app:$APP = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+defineDebugInfoPackage lightsoff \
+	$instDir/$APP
+
+BUILD()
+{
+	make -C src OBJ_DIR=objects $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $instDir $instDir/docs $instDir/media
+	cp src/objects/$APP $instDir
+	cp -r docs $instDir
```

Comment:
With the current recipe we have these files/dirs in `$appsDir/LightsOff/`:
```
apps/LightsOff/LightsOff
apps/LightsOff/docs/
apps/LightsOff/docs/GettingStarted.html
apps/LightsOff/media/
apps/LightsOff/media/altwin.wav
apps/LightsOff/media/click.wav
apps/LightsOff/media/fanfare.wav
```
I think it would be better to put the html doc in `$docDir` and the wav media in `$dataDir/LightsOff/`.

**Pros:**
* The html file would be in the recommended directory tree.
* The wav files would be in a directory meant to be used for data files.
* The app would be launchable from the command line with a simple `LightsOff` since `/boot/system/apps` is in the `PATH`.

**Cons:**
* This might need some extra changes in the sources.

That said, I don't object a merge as is.

BTW, it would be nice to drop the old recipe, `lightsoff-1.0.recipe`.